### DISCUSSION TITLE
Beta version

### DISCUSSION THREAD

**Discussion by amansarroha** [mbentley/docker-omada-controller#502] [Help] [answered]
After downloading 5.15.6.7 it still shows 5.15.6.4 in the server settings page.

**Comment by mbentley** [comment, marked as answer]
When changing the `install.sh`, the builds can take a while - the betas are still re-building now.

*edit: they're done now.

**Reply by amielgb** [reply]
is anti sharing enalbled already?

**Reply by mbentley** [reply]
I don’t know what anti sharing is.

**Reply by avast1211** [reply]
I think he/she mean anti-tethering or anti Wi-Fi sharing for business I guess

**Reply by mbentley** [reply]
In that case, I don't know.  The controller will have whatever settings are the default or what has been already set if upgrading.

**Comment by avast1211** [comment]
I think it fixed now I'm experiencing errors such as "This client does not exist" or "Omada server error on Omada APP

**Reply by mbentley** [reply]
Where are you seeing these errors?  They're not errors I am familiar with.

**Reply by avast1211** [reply]
In omada apps both android and iOS
![a002049f5b484a0ab267c79d14e61cfc](https://github.com/user-attachments/assets/38e966f2-f90c-4a0c-bc2b-59e82ceb9a28)
![58f2fa3941cc445f9c995b52306a360b](https://github.com/user-attachments/assets/091a4d57-f3ff-4356-a997-e76181b34a5d)
